data/wanderers middle.txt

```diff
@@ -3039,8 +3025,8 @@ event "wanderers: sestor shutdown"
 		fleet "Small Kor Sestor" 3200
 		fleet "Large Kor Sestor" 4800
 	system "Mesuket"
-		fleet "Large Kor Mereti" 900
-		fleet "Large Kor Sestor" 10000
+		remove fleet "Large Kor Sestor"
+		add fleet "Large Kor Sestor" 10000
```

Comment:
If we change how add/remove works for preexisting fleets ( #2742), this subsection (and L3137) should be revisited too.